docs/administrative-requirements.md

```diff
@@ -106,25 +101,50 @@ Resources:
 * [PIA Training Resources (VA SharePoint, must be on VA network to view)](https://dvagov.sharepoint.com/sites/OITPrivacyHub/SitePages/Privacy-Impact-Assessment.aspx?csf=1&web=1&e=u8YJ0w)
```

Comment:
https://dvagov.sharepoint.com/sites/OITPrivacyHub/SitePages/PIA-Training-Resources.aspx goes directly to the Training Resources page.

Reply:
Updated, and flipped the order so PTA is first.